### PR TITLE
ZEN-28191: Infrastructure device overview datetimes

### PR DESCRIPTION
**Pull request by lundybernard**
Fixes ZEN-28191

### CODE REVIEW

File: Products/Zuul/infos/device.py [hunk 1]
```diff
@@ -151,10 +158,13 @@ def _allSeverities(self):
...
-            self._severities = dict(
-                (uuid, self._eventFacade.getSeverityName(severity).lower())
-                    for (uuid, severity) in self._eventFacade.getWorstSeverity(uuids, ignore=(SEVERITY_INFO, SEVERITY_DEBUG)).iteritems()
-            )
+            self._severities = {
+                uuid: self._eventFacade.getSeverityName(severity).lower()
+                for (uuid, severity)
+                in self._eventFacade.getWorstSeverity(
+                    uuids, ignore=(SEVERITY_INFO, SEVERITY_DEBUG)
+                ).iteritems()
```
Comment (jpeacock-zenoss) [inline Products/Zuul/infos/device.py]: There's a few things going on in here that are a bit wordy.  For legibility, perhaps
```
    events = self._eventFacade.getWorstSeverity(
        uuids, ignore=(SEVERITY_INFO, SEVERITY_DEBUG)
    )
    self._severities = {
        uuids: self._eventFacade.getSeverityName(severity).lower()
        for (uuid, severity) in events.iteritems()
    }
```
as a suggestion.

Comment (lundybernard) [reply Products/Zuul/infos/device.py]: Fixed as recommended

**Comment by lundybernard** [comment]
This fix has been merged into the develop branch, it will not be backported to support/5.3.x due to the extensive testing requirements.

Closing this PR
